src/js/components/shepherd-modal.svelte

```diff
@@ -120,15 +127,23 @@
    * @private
    */
   function _styleForStep(step) {
-    const { modalOverlayOpeningPadding, modalOverlayOpeningRadius } = step.options;
+    const {
+      modalOverlayOpeningPadding,
+      modalOverlayOpeningRadius
+    } = step.options;
 
-    if (step.target) {
+    if (step) {
```

Comment:
Do we need the if/else here at all anymore? I think `step` has to be defined to get in this method at all.